.github/workflows/script/sw_build_and_push_gem.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+GEM_NAME="$1"
+
+case $GEM_NAME in
```

Comment:
Nice that we are able to have a map here 👍 compare to https://github.com/solarwinds/opentelemetry-ruby-contrib/pull/1